helm/chart-operator-resource-chart/templates/chartconfig.yaml

```diff
@@ -1,7 +1,8 @@
-apiVersion: "provider.giantswarm.io/v1alpha1"
+apiVersion: "core.giantswarm.io/v1alpha1"
 kind: ChartConfig
 metadata:
   name: "{{.Values.chart.name}}"
+  namespace: "kube-system"
```

Comment:
I miss the big picture here. Why is there a chart config in the helm dir? This means usually the chart config is deployed but then I wonder why. The CR usually says an operator has to reconcile something. Based on the values file it means the operator is reconciling itself. How does that work?

Reply:
We usually use a resource chart for e2e testing of operators, awsconfig for aws-operator, certconfig for cert-operator, etc. These resource charts are only used by the tests that live in the operator repos (they belong to the operator repos), but before recent changes in architect, we only could have one chart per helm dir. So, in this case, we have in the helm dir the operator chart and the resource chart used for testing.

Reply:
I see. There are multiple dirs in the helm dir. That makes sense.